.github/workflows/e2e-latest.yml

```diff
@@ -41,6 +42,16 @@ jobs:
                        --tag=$GITHUB_SHA
       env:
         PROJECT_ID: ${{ secrets.E2E_PROJECT_ID  }}
+    - name: Build Load Generator
+      run: |
+        set -x
+        gcloud auth configure-docker --quiet
+        # push latest load generator to test image repository
+        IMAGE_PATH=gcr.io/$PROJECT_ID/gcr.io/stackdriver-sandbox-230822/sandbox/loadgenerator/basic:$GITHUB_SHA
```

Comment:
Sorry, wrong path. This should be:

`IMAGE_PATH=gcr.io/$PROJECT_ID/tests/gcr.io/stackdriver-sandbox-230822/sandbox/loadgenerator/basic:$GITHUB_SHA`